include/debug.h

```diff
@@ -28,6 +28,9 @@ enum {
 	LOG_LEVEL_ERROR,
 	LOG_LEVEL_WARNING,
 	LOG_LEVEL_INFO,
+#if TEST
```

Comment:
This is what we want, but please put this after `DEBUG`.

Reply:
Also, this part (and _only_ this part and `debug.c`) needs to be defined unconditionally, or things break in mysterious ways when only part of the files are built with `TEST=1`

Reply:
what do you mean

Reply:
Don't wrap `LOG_LEVEL_TEST` with `#if TEST`. Don't wrap the entry in `prefixes` with `#if TEST`.

Reply:
what is "everything else"?

Reply:
...everything else in the PR?
- The entire contents of `common/test.c` after the `#include` statements
- The other macros in `include/debug.h`
- The contents of `include/common/test.h` inside the header guards and after any `#include` statements
- The function call in `common/main.c`
- Anything else I forgot